core/services/feeds/orm.go

```diff
@@ -541,6 +552,35 @@ SELECT exists (
 	return exists, errors.Wrap(err, "JobProposalSpecVersionExists failed")
 }
 
+// DeleteProposal performs a soft delete of the job proposal by setting the
+// status to deleted, remove the external job id, and update the status to
+// deleted
+func (o *orm) DeleteProposal(id int64, qopts ...pg.QOpt) error {
+	stmt := `
+UPDATE job_proposals
+SET status = $1,
+	external_job_id = $2,
+	pending_update = TRUE,
```

Comment:
`pending_update` should be set to false

Reply:
I set the pending_update to true here because they need to go cancel the job